tests/x11/libqt5_qtbase.pm

```diff
@@ -21,6 +21,13 @@ use version_utils 'is_sle';
 use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
 
 sub run {
+    if (is_sle) {
+        select_console 'root-console';
```

Comment:
you can do this in the serial console

Reply:
okay, I will change and push new commit.